gen/atlas.tmpl

```diff
@@ -11,33 +14,29 @@ permissions:
   contents: read
   pull-requests: write
 jobs:
-  lint:
+  atlas:
     {{- template "services" . }}
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
-      - uses: ariga/atlas-action@v0
+      - uses: ariga/setup-atlas@v0
         with:
-          dir: '{{ .Path }}'
-{{- with .DirName }}
+          cloud-token: {{`${{ secrets.`}}{{ .SecretName }}{{` }}`}}
+      - uses: ariga/atlas-action/migrate/lint@master
+        with:
+          dir: 'file://{{ .Path }}'
+          {{- with .DirName }}
           dir-name: '{{ . }}'
-{{- end }}
+          {{- end }}
           {{ template "UseServices" (args .Driver "localhost") }}
-          cloud-token: {{`${{ secrets.`}}{{ .SecretName }}{{` }}`}}
-  sync:
-    needs: lint
-    {{- template "services" . }}
-    if: github.ref == 'refs/heads/{{ .DefaultBranch }}'
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - uses: ariga/atlas-sync-action@v0
+      - uses: ariga/atlas-action/migrate/push@master
```

Comment:
Should be using `v1` by default

```suggestion
      - uses: ariga/atlas-action/migrate/push@v1
```

Reply:
didn't we agree on master, and that it will fetch the latest?

Reply:
I think that is better to specify explicit version, but not crucial IMO

Reply:
as far as I understood from my conversations with Rotem, using v{major} is the way to go for pulling latest action on github actions ecosystem (you will automatically get minor versions, but nothing breaking)